openlibrary/templates/books/edit/edition.html

```diff
@@ -208,8 +208,9 @@
                                     $for role in edition_config.roles:
                                         <option>$role</option>
 
-                                    <!-- <option>---</option> -->
-                                    <!-- <option value="__add__">$_("Add a new role")</option> -->
+                                    $if ctx.user and (ctx.user.is_admin() or ctx.user.is_librarian()):
```

Comment:
```suggestion
                                    $if is_privileged_user:
```
https://github.com/internetarchive/openlibrary/blob/a9d82b6bddf8e6a3c86c4bede084ed2c6ad607ec/openlibrary/templates/books/edit/edition.html#L4

This will DRY things up.